### PR TITLE
Fix / layout of invoices on mobile

### PR DESCRIPTION
**Pull request by MelissaDTH**
## This PR

- Ensures users can always access the invoice button on mobile, by adjusting the layout slightly || **OTT-861**
  - Has been approved by the designer

Screenshot 2024-02-20 at 17 38 28

### CODE REVIEW

**Review by ChristiaanScheermeijer — APPROVED** [review]
Nice fix @MelissaDTH 😄

**Review by langemike — APPROVED** [review]
Nice!

**Review by royschut — APPROVED** [review]
Nice @MelissaDTH!

Two minor comments:
 - When implementing a fix, the branch name is not a feat, but could be `fix/invoice-mobile` :) 
 - Shouldn't there be a small margin between the buttons?

**Comment by MelissaDTH** [comment]
@royschut 

> 
> * When implementing a fix, the branch name is not a feat, but could be `fix/invoice-mobile` :)
> * Shouldn't there be a small margin between the buttons?

* Yes I'm aware, I quickly created a branch to push my changes to, forgetting I could've created a `fix` branch
* That missing margin has been in the app for a very long time and is regarding the buttons. So it isn't in the scope of my ticket, I will quickly see if I can find a robust fix but will keep time for that to a minimum! ;) 
   * **Update**: I've applied a small margin between these buttons, there can be a situation where the second button 'Update payment details' is not rendered, so we only apply the `8px` margin when the button is present

File: packages/ui-react/src/components/Payment/Payment.tsx [hunk 1]
```diff
@@ -315,7 +315,12 @@ const Payment = ({
                   autoComplete="cc-csc"
                 />
               </div>
-              <Button label={t('account:payment.edit_card')} variant="outlined" onClick={onEditCardDetailsClick} />
+              <Button
+                className={`${canUpdatePaymentMethod ? styles.editCard : ''}`}
```
Comment (royschut) [inline packages/ui-react/src/components/Payment/Payment.tsx]: Minor, but classNames can do such things for us:

```javascript
className={classNames(canUpdatePaymentMethod && styles.editCard)}
```
or 
```jsx
className={classNames({ [styles.editCard] : canUpdatePaymentMethod })}
```

**Comment by royschut** [comment]
> * **Update**: I've applied a small margin between these buttons, there can be a situation where the second button 'Update payment details' is not rendered, so we only apply the `8px` margin when the button is present

Nice that you could add this small improvement as well. :) 
Merge?

**Pull request merged**